mars/services/task/execution/ray/executor.py

```diff
@@ -360,6 +380,19 @@ async def get_progress(self) -> float:
 
     async def cancel(self):
         """Cancel execution."""
+        subtask_num = len(self._output_object_refs_nums)
+        if subtask_num > 0:
+            self._cancelled = True
+            pos = 0
+            for i in range(0, subtask_num):
+                if i > 0:
+                    pos += self._output_object_refs_nums[i - 1]
+                await _cancel_ray_task(
+                    self._cur_stage_output_object_refs[pos],
+                    self._config.get_execution_config()
+                    .get("ray", {})
+                    .get("subtask_cancel_timeout", 3),
```

Comment:
Move this code to the `RayExecutionConfig` in ray/config.py

Reply:
Add a method called `get_subtask_cancel_timeout` in `RayExecutionConfig`?

Reply:
Either property or method are OK.